### PR TITLE
cmd: `humility halt` and `humility resume`

### PR DESCRIPTION
**Pull request by beezow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1216,7 +1224,10 @@ VSC8552; this is indicated with `--` in the relevant table positions.
 
 ### `humility openocd`
 
-This command launches OpenOCD based on the config file in a build archive
+This command launches OpenOCD based on the config file in a build archive,
+which then allows one to connect with either GDB or directly via telnet.
+If the intention is to only run GDB, note that `humility gdb --run-openocd`
+will both run OpenOCD and run a foreground GDB that is connected to it.
```
Comment (furquan-rivos) [inline README.md:1230]: unrelated change

Comment (beezow) [reply README.md:1230]: Yea, it seems that the base readme hadn't been regenerated in a bit.  Upstream has had the same problem on some of their recent PRs.
I don't know if it is worth spinning into its own PR, but always can.

Comment (furquan-rivos) [reply README.md:1230]: No, I think this is fine.

**Review by furquan-rivos — APPROVED** [review]
